msl_expressions/autogenerated/src/Plans/Attack/Dribble1434049476066.cpp

```diff
@@ -84,7 +84,8 @@ namespace alicaAutogenerated
             return false;
         }
         double distance = numeric_limits<double>::max();
-        auto obs = wm->obstacles->getEgoVisionObstaclePoints();
+        //better clustered opps?
+        auto obs = wm->obstacles->getAlloObstaclePoints();
         auto ballPos = wm->ball->getEgoBallPosition();
```

Comment:
Changed from Ego to Allo because a few lines later there is a check whether the obs points are inside an allo-centric triangle. Same in PassPlay plan.